src/main/resources/programs/routine.json

```diff
@@ -0,0 +1,11 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "3390ea92-18bb-480f-a2f0-fff7850def4b",
+  "item": [
+    {
+      "linkId": "30cdb1dc-1548-4113-a463-6992f7110396",
```

Comment:
@HerbertYiga what happens if we re-use the same concept as [this](https://github.com/DIGI-UW/OpenELIS-Global-2/blob/develop/src/main/resources/programs/pathology.json#L2599C20-L2599C56)
Could you also ensure that all programs have this question in their additional questionnaires

Reply:
sure ,updated and a screen shot added here 
<img width="1410" height="865" alt="Screenshot 2025-10-28 at 02 11 29" src="https://github.com/user-attachments/assets/b8c6bc89-851a-41d3-b224-52dd5bd3277b" />